app/templates/orders/order_block.hbs

```diff
@@ -1,31 +1,82 @@
-<li>
+<li class="order_block">
 
-    <div class="row state_details">
-      <div class="small-5 columns">
-        {{order.code}} - {{order.status}}
-      </div>
-      <div class="small-4 columns">
-        {{order.designatedOrdersPackages.length}} <i class="fa fa-shopping-basket"></i>
-        {{order.dispatchedOrdersPackages.length}} <i class="fa fa-ship"></i>
+    <div class="row order_header">
+
+      <div class="small-4 columns order_transport"> 
+        <i class="fa fa-lg fa-{{ order.transportIcon }}"></i> {{ order.transportLabel }}
       </div>
-      <div class="small-3 columns">
-        {{display-datetime order.createdAt format="DD MMM 'YY"}}
+
+      <div class="small-3 columns order_code">
+        {{order.code}}
       </div>
-    </div>
 
-    <div class="row">
-      <div class="small-12 columns">
-        {{order.organisation.name}}
+      <div class="small-5 columns order_state">
+        <div class="order_state_icon {{ order.state }}">
+          <i class="fa fa-{{ order.stateIcon }}"></i>
+        </div>
+        <div class="order_state_text"> {{ t (concat "order." order.state) }} </div>
       </div>
+
     </div>
 
-    <div class="row">
-      <div class="small-12 columns">
-        {{order.stockitContact.fullName}}: {{order.stockitContact.displayNumber}}
-        {{#if order.localOrder.clientName}}
-          for {{order.localOrder.clientName}}
-        {{/if}}
-      </div>
+    <div class="row order_body">
+
+      {{#if order.orderTransport.scheduledAt }}
```

Comment:
@patrixr , correct me if I'm wrong. For StockIt Orders, OrderTransport doesn't exist right? So it'll fail for StockIt orders. So I think we can add a check for that here. Unfortunately Handlebars doesn't support `and | or` or any other conditions with if so we've to write helpers for such cases. I think you can use the below line

`{{#if (is-and order.orderTransport order.orderTransport.scheduledAt)}}`

Here we've used `is-and` helper which you can find in **goodcity-lib** repo.

Reply:
@Darshan-Chauhan I was under the impression handlebars handled undefined variables. I've tested it here http://jsfiddle.net/x90Lbe38/2/

I'll switch to is-and to be safe, but it'd be good to check it out :)